src/mumble/GlobalShortcut_win.h

```diff
@@ -29,6 +29,8 @@ class GlobalShortcutWin : public GlobalShortcutEngine {
 	Q_DISABLE_COPY(GlobalShortcutWin)
 
 public:
+	static const size_t QUEUE_CAPACITY = 64;
```

Comment:
```suggestion
	static constexpr size_t QUEUE_CAPACITY = 64;
```
We might as well… :D